src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs

```diff
@@ -11,7 +11,6 @@
 
 namespace System.Net.Http.Functional.Tests
 {
-   [ActiveIssue(29802, TargetFrameworkMonikers.Uap)]
    public abstract class HttpClientHandler_ResponseDrain_Test : HttpClientTestBase
```

Comment:
Should we rename the test class something like: `SocketsHttpHandler_ResponseDrain_Test`?

Reply:
Maybe in the future, this class should be renamed/merged with the non-abstract class we already have:

https://github.com/dotnet/corefx/blob/b46045f9132597e1600e3ceafd6bbeb272576a62/src/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs#L132

But I didn't want to refactor everything yet in this PR.

Reply:
Ok.